Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -161,7 +161,8 @@ bool GameFile::TryLoadVolume()
 
   m_game_id = QString::fromStdString(volume->GetGameID());
   std::string maker_id = volume->GetMakerID();
-  volume->GetTitleID(&m_title_id);
+  if (std::optional<u64> title_id = volume->GetTitleID())
+    m_title_id = *title_id;
```

Comment:
We should probably do that, though it might be better to leave it for a separate PR.